doc/Development-Guide-Rad-Tool.md

```diff
@@ -0,0 +1,23 @@
+### Introduction
+
+ In this document, we will introduce **ASP.NET Zero Power Tool** and expalin it.
+
+### Purpose Of The Tool
+
+This tool is developed to minimize the effort of creating a new table. It creates all related layers (including UI) by defining an entity.
+
+### Download And Install
+
+ If your project version is 5.1.0+, all you have to do is just installing **ASP.NET Zero Power Tools** extension on visual studio, from https://marketplace.visualstudio.com/items?itemName=Volosoft.AspNetZeroPowerTools.
+
+ If your project version is below 5.1.0, you also have to copy the AspNetZeroRadTool folder to your own project, from a newly downloaded 5.1.0+ project.
+ 
+### How To Use It?
+ 
+ Extension is inside **Tools** menu (tools-> Asp.Net Zero -> Create An Entity). When you run it, you will see the interface for creating an entity. After carefully filling the fields, press **Generate** button to start the code generation process. 
+ 
+ A simple console will appear and give you information about the process. If there is no warning and fail, run your project to see the results (Angular developers will have to run nswag manually). You probably won't see a new page, but don't worry and give yourself a **permission**.
+ 
```

Comment:
Add a warning: Be sure that you have saved your works before running this tool, since it will add new files and modify some of the existing files. We strongly recommend to use a source control system (like Git). Otherwise, backup your project.

Reply:
Add a short list of "what it generates". Like:

* Creates the entity class
* Adds entity to EF Core dbcontext
* Add a new migration (optional)
* Creates application service and DTOs for the entity
* ...and so on.